app/templates/download.html

```diff
@@ -0,0 +1,27 @@
+{% from 'macros/page_heading.html' import page_heading %}
```

Comment:
I vote for one page. And use [browse list](http://alphagov.github.io/digitalmarketplace-frontend-toolkit/browse-list.html) for the list of frameworks.